tests/date/bad/uncomparable_duration.catala_en

```diff
@@ -42,83 +42,55 @@ scope Ge:
 
 ```catala-test-inline
 $ catala test-scope Ge
-[ERROR] Cannot compare together durations that cannot be converted to a
-  precise number of days
+[ERROR] Error during evaluation: comparing durations in different units (e.g.
+  months vs. days).
 
-┌─⯈ tests/date/bad/uncomparable_duration.catala_en:40.23-40.30:
+┌─⯈ tests/date/bad/uncomparable_duration.catala_en:40.31-40.33:
 └──┐
 40 │   definition d equals 1 month >= 2 day
-   │                       ‾‾‾‾‾‾‾
-   └┬ `UncomparableDurations` exception management
-    └─ `>=` operator
```

Comment:
So the runtime exceptions do not carry a law heading breadcrumb anymore. Maybe we should add law heading breadcrumbs to `Runtime.position`?

Reply:
They do, they do, it's already there.